code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -215,7 +215,7 @@
 	stat_allowed = 1
 
 /obj/effect/proc_holder/spell/vampire/mob_aoe/glare/cast(list/targets, mob/user = usr)
-	user.visible_message("<span class='warning'><b>[user]'s eyes emit a blinding flash!</span>")
+	user.visible_message("<span class='warning'><b>[user]'s eyes emit a blinding flash!</b></span>")
```

Comment:
Are the `<b>` tags needed here? I thought `span class = 'warning'` already made a bold message.